code/modules/mob/living/carbon/human/species_types/podpeople.dm

```diff
@@ -10,6 +10,8 @@
 	burnmod = 1.25
 	heatmod = 1.5
 	meat = /obj/item/weapon/reagent_containers/food/snacks/meat/slab/human/mutant/plant
+	disliked_food = VEGETABLES | FRUIT | GRAIN //They think plant murder is wrong.

```

Comment:
How ironic, botanists who can't eat their own fruits.
I'd go on a different direction, considering biology instead of morals it'd make sense to put plants as liked and nonplants as toxic or at least disliked. I know they can 'eat' with light, but podpeople are 90% botanists and making them puke for eating their work is pretty counterintuitive.